src/androidTest/java/androidx/graphics/PathTest.kt

```diff
@@ -152,4 +152,23 @@ class PathTest {
 
         assertEquals(RectF(0.0f, 0.0f, 15.0f, 15.0f), r)
     }
+
+    @Test fun testPathFromPoints() {
+        val rect = RectF(0f, 0f, 40f, 40f)
+
+        val points = ArrayList<PointF>()
+        points.add(PointF(rect.left, rect.top))
+        points.add(PointF(rect.right, rect.top))
+        try {
+            val illegalPath = createPathFromPoints(points)
```

Comment:
I don't understand what makes such a path "illegal"

Reply:
At first I thought it should draw a contour by canvas, not a line. This is not necessarily illegal